src/main/java/laddergame/domain/LadderConnectedDirection.java

```diff
@@ -0,0 +1,34 @@
+package laddergame.domain;
+
+import java.util.function.Function;
+
+public enum LadderConnectedDirection {
+
+    RIGHT(position -> position + 1),
+    LEFT(position -> position - 1),
+    NONE(position -> position);
+
+    private final Function<Integer, Integer> nextLinePositionCalculation;
```

Comment:
같은 타입을 반환하는 Function의 경우 UnaryOperator를 활용해 보시는 것도 좋을 것 같아요 😃 
```suggestion
    private final UnaryOperator<Integer> nextLinePositionCalculation;
```